core/src/main/java/org/kiwix/kiwixmobile/core/settings/CorePrefsFragment.java

```diff
@@ -84,13 +92,23 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
       .build()
       .inject(this);
```

Comment:
delete the injection here, it wasn't using the cached component anyway

Reply:
@macgills I tried to remove this line but it crashes.

Reply:
Post the crash

Reply:
<description>
<summary>
Logs
</summary>

```
2020-11-17 15:38:32.230 25989-25989/org.kiwix.kiwixmobile E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.kiwix.kiwixmobile, PID: 25989
    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String org.kiwix.kiwixmobile.core.utils.SharedPreferenceUtil.getPrefStorage()' on a null object reference
        at org.kiwix.kiwixmobile.settings.KiwixPrefsFragment.setStorage(KiwixPrefsFragment.kt:43)
        at org.kiwix.kiwixmobile.core.settings.CorePrefsFragment.onCreatePreferences(CorePrefsFragment.java:96)
        at org.kiwix.kiwixmobile.settings.KiwixPrefsFragment.onCreatePreferences(KiwixPrefsFragment.kt:37)
        at androidx.preference.PreferenceFragmentCompat.onCreate(PreferenceFragmentCompat.java:160)
        at androidx.fragment.app.Fragment.performCreate(Fragment.java:2685)
        at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:280)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1187)
```
</description>